algos/index.rst

```diff
@@ -13,7 +13,10 @@ build pipelines.
 
    "Asynchronous sample rate conversion", "Converts between common sample rates and connects pipelines with different clock domains", "Yes", "Xtensa HiFi3", "Upstream"
    "Channel selector", "Copies the selected channel from the source buffer to the sink buffer", "Yes", "N/A", "Upstream"
+   "Crossover", "Splits audio up into at most 4 different bands for individual processing", "Yes", "No", "Planned"
+   "DCBlocker", "Simpled highpass filter to remove DC components from audio", "Yes", "N/A", "Planned"
```

Comment:
Is "simpled" a typo?

Reply:
@cujomalainey If "simpled" is a typo, what is the correct word?

Reply:
Simple, just accidentally added a d

Reply:
Will correct in a separate PR.